containers/podman-desktop
# chore: update to docusaurus v2.0 (instead of beta)

### What does this PR do?
https://docusaurus.io/blog/2022/08/01/announcing-docusaurus-2.0
v2 is now GA, switch to it instead of the beta
it is still using React 17 (so we can't bump React to v18)

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

- dev mode:  run `yarn start` command in `website` subdirectory and check everything is working as expected
- production mode: run `yarn run build` and then `yarn run serve` and check pages are also working as expected

Change-Id: I728789a9c175f7c75bc95de4e8215aad9f0e40b6
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
